From b18b3dc51a93e12172d22c6b8bb92d32b6e8ebb0 Mon Sep 17 00:00:00 2001
From: Carlos Garcia Campos <carlosgc@gnome.org>
Date: Sun, 17 Jul 2016 10:46:27 +0200
Subject: ev-toolbar: Fix ev_toolbar_has_visible_popups after cb3d4b2

The toolbar was changed to use popovers for the view and actions menus,
but ev_toolbar_has_visible_popups was still checking if the actions had
a GtkMenu visible.
---
 shell/ev-toolbar.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/shell/ev-toolbar.c b/shell/ev-toolbar.c
index eff5a08..fdc3fc1 100644
--- a/shell/ev-toolbar.c
+++ b/shell/ev-toolbar.c
@@ -318,19 +318,19 @@ ev_toolbar_new (EvWindow *window)
 gboolean
 ev_toolbar_has_visible_popups (EvToolbar *ev_toolbar)
 {
-        GtkMenu          *popup_menu;
+        GtkPopover       *popover;
         EvToolbarPrivate *priv;
 
         g_return_val_if_fail (EV_IS_TOOLBAR (ev_toolbar), FALSE);
 
         priv = ev_toolbar->priv;
 
-        popup_menu = gtk_menu_button_get_popup (GTK_MENU_BUTTON (priv->view_menu_button));
-        if (gtk_widget_get_visible (GTK_WIDGET (popup_menu)))
+        popover = gtk_menu_button_get_popover (GTK_MENU_BUTTON (priv->view_menu_button));
+        if (gtk_widget_get_visible (GTK_WIDGET (popover)))
                 return TRUE;
 
-        popup_menu = gtk_menu_button_get_popup (GTK_MENU_BUTTON (priv->action_menu_button));
-        if (gtk_widget_get_visible (GTK_WIDGET (popup_menu)))
+        popover = gtk_menu_button_get_popover (GTK_MENU_BUTTON (priv->action_menu_button));
+        if (gtk_widget_get_visible (GTK_WIDGET (popover)))
                 return TRUE;
 
         if (ev_zoom_action_get_popup_shown (EV_ZOOM_ACTION (ev_toolbar->priv->zoom_action)))
-- 
cgit v0.12

